app/views/layouts/application.html.erb

```diff
@@ -54,6 +54,7 @@
           { name: "Home", url: root_path },
           ({ name: "Draft trainees", url: drafts_path, current: active_link_for("drafts", @trainee) } if can_view_drafts?),
           { name: "Registered trainees", url: trainees_path, current: active_link_for("trainees", @trainee) },
+          { name: "Reports", url: reports_path },
```

Comment:
is the report link hidden from lead school users?

Reply:
should be fixed now